backend/__init__.py

```diff
@@ -9,10 +9,21 @@
 from flask_restful import Api
 from flask_sqlalchemy import SQLAlchemy
 
-
 from backend.config import EnvironmentConfig
 
 
+def sentry_init():
+    """Initialize sentry.io event tracking"""
+    import sentry_sdk
+    from sentry_sdk.integrations.flask import FlaskIntegration
+
+    sentry_sdk.init(
```

Comment:
@dakotabenjamin is [NetworkEnvironment](https://github.com/hotosm/tasking-manager/blob/feature/sentry-backend-integration/scripts/aws/cloudformation/tasking-manager.template.js#L7) available as a ENV VAR to the backend?
To set the environment on sentry, we just need to add `environment=foo` here

Reply:
can we use the stack naming convention for that, similar to how we do New Relic? https://github.com/hotosm/tasking-manager/blob/feature/sentry-backend-integration/scripts/aws/cloudformation/tasking-manager.template.js#L394

Reply:
Yes, could we change the env var from `NEW_RELIC_ENVIRONMENT` to `TM_ENVIRONMENT`, so if we stop using NewRelic, we don't need to change the env var and make the variable name more expressive.

Reply:
`NEW_RELIC_ENVIRONMENT` is something used internally by New Relic, so we can't change it.